recipes/pcre2/all/conandata.yml

```diff
@@ -1,16 +1,27 @@
 sources:
   "10.32":
-    url: "https://ftp.pcre.org/pub/pcre/pcre2-10.32.tar.gz"
-    sha256: "9ca9be72e1a04f22be308323caa8c06ebd0c51efe99ee11278186cafbc4fe3af"
+    url:
+      - "https://sourceforge.net/projects/pcre/files/pcre2/10.32/pcre2-10.32.tar.bz2" 
```

Comment:
It won't work if the checksums are different. If the first link fails, the second has a different sah256 which Conan won't allow.

Reply:
I see, you changed the tar file to bz2.  Nice!